### PR TITLE
Special action log

### PR DESCRIPTION
**Pull request by RobNadal**

### CODE REVIEW

File: gosushi_client/src/components/Board.jsx [hunk 1]
```diff
@@ -134,7 +135,7 @@ const Board = ({ socket }) => {
           onClick={handleFinishTurn}
         >
           Finish Turn
-          <span class="hovertext">You cannot undo this action</span>
+          <span className="hovertext">You cannot undo this action</span>
```
Comment (selinali2010) [inline gosushi_client/src/components/Board.jsx:138]: Loool I fixed this in my PR too

File: gosushi_client/src/components/SpecActionsLog.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          break;
+        case 'Menu':
+          details = <span>, ordering <b className={cCardCN}>{entry.chosenCard}</b> from the deck.</span> 
+          break;
+        case 'Special Order':
+          details = <span>, duplicating their <b className={cCardCN}>{entry.chosenCard}</b>.</span> 
+          break;
+        case 'Spoon':
+          if (entry.stolenFromPlayer) {
+            details = <span>, stealing a(n) <b className={cCardCN}>{entry.chosenCard}</b> from <b>{entry.stolenFromPlayer}</b>'s hand'.</span>
```
Comment (selinali2010) [inline gosushi_client/src/components/SpecActionsLog.jsx]: I think there's an extra apostrophe (') after the word "hand".
image

File: gosushi_client/src/index.scss [hunk 3]
```diff
@@ -10,6 +12,29 @@
...
+  --onigiri: #d42e93;
+  --misosoup: #5c9995;
+  --sashimi: #c8d92b;
+  --tempura: #c098c6;
+  --tofu: #6aa141;
+  --chopsticks: #a1d6cd;
+  --spoon: #4e5151;
+  --menu: #dd4f4a;
+  --takeoutbox: #cba178;
+  --soysaunce: #f49759;
```
Comment (selinali2010) [inline gosushi_client/src/index.scss]: "soysaunce" -> "soysauce"

File: gosushi_client/src/components/SpecActionsLog.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+.log-onigiri { color: var(--onigiri); }
+.log-misosoup { color: var(--misosoup); }
+.log-sashimi { color: var(--sashimi); }
+.log-tempura { color: var(--tempura); }
+.log-tofu { color: var(--tofu); }
+.log-chopsticks { color: var(--chopsticks); }
+.log-spoon { color: var(--spoon); }
+.log-menu { color: var(--menu); }
+.log-takeoutbox { color: var(--takeoutbox); }
+.log-soysaunce { color: var(--soysaunce); }
```
Comment (selinali2010) [inline gosushi_client/src/components/SpecActionsLog.scss]: "soysaunce" -> "soysauce"

**Review by selinali2010 — CHANGES_REQUESTED** [review]

**Review by hannahli2010 — APPROVED** [review]

**Review by selinali2010 — APPROVED** [review]

**Pull request merged**